exonum/src/node/mod.rs

```diff
@@ -950,8 +951,14 @@ impl Node {
             node_cfg.service_keypair(),
             ApiSender::new(channel.api_requests.0.clone()),
         );
-        let blockchain = BlockchainBuilder::new(blockchain, node_cfg.consensus.clone())
-            .with_rust_runtime(channel.endpoints.0.clone(), services)
+        let (rust_runtime, genesis_config) = create_rust_runtime_and_genesis_config(
+            channel.endpoints.0.clone(),
+            node_cfg.consensus.clone(),
+            services,
+        );
+
+        let blockchain = BlockchainBuilder::new(blockchain, genesis_config)
+            .with_runtime(rust_runtime)
             .with_external_runtimes(external_runtimes)
```

Comment:
This method looks like an redundant.

Reply:
Going to get rid of it during the `Node::new()` refactoring.